### PR TITLE
Add support for umami tracking

### PR DESCRIPTION
**Pull request by michaelcontento**
Hi,

I started to use [umami](https://umami.is/) as analytics solution and thus added a very simple integration to your theme.

Feel free to tinker / tweak / improve my solution, as I'm quite new to the world of hugo theme development 😅

### CODE REVIEW

**Comment by igor-baiborodine** [comment]
@michaelcontento, I'm hesitant to add any web analytics integration other than Google Analytics which currently dominates the web analytics market. Also, the integration with Google Analytics is built-in in Hugo. So, in my opinion, any other web analytics integration should be done either via overriding templates provided by the Bilberry Hugo theme, e.g., `layouts/_default/baseof.html`, or via adding a special file, for [example](https://github.com/igor-baiborodine/kiroule.com/blob/master/static/yandex_0d1febf3177d5cbd.html), as I did for the integration with Yandex Webmaster on my blog kiroule.com. @Lednerb, what's your opinion on that?

**Comment by Lednerb** [comment]
In general I'm not a big friend of user tracking and site analytic tools.

(btw. The theme was once developed to work even without any javascript).

There are some good alternatives to Google Analytics and it would be nice if more users will switch to more privacy-respecting tools and services.

However I think maintaining too much external services that are not in use by a lot of users or are not requested frequently will prevent us from improving the theme for the general user base.

-----

**TL;DR**
As @igor-baiborodine suggested, overwriting the default layout file should be the best of both worlds: practical enough (typically just changing the file once) and it saves a lot of time for testing / fixing and maintaining the theme.
